platform-workflow-module/src/main/java/com/softicar/platform/workflow/module/workflow/transition/WorkflowTransitionActionExecutor.java

```diff
@@ -19,18 +23,57 @@ public class WorkflowTransitionActionExecutor {
 	private final AGWorkflowItem item;
 	private final AGWorkflowTransition transition;
 	private final AGUser user;
+	private INullaryVoidFunction afterExecutionCallback;
 
 	public WorkflowTransitionActionExecutor(AGWorkflowItem item, AGWorkflowTransition transition, AGUser user) {
 
 		this.item = Objects.requireNonNull(item);
 		this.transition = Objects.requireNonNull(transition);
 		this.user = Objects.requireNonNull(user);
+		this.afterExecutionCallback = INullaryVoidFunction.NO_OPERATION;
+	}
+
+	public WorkflowTransitionActionExecutor setAfterExecutionCallback(INullaryVoidFunction afterExecutionCallback) {
```

Comment:
I think we should rename this to `setCallbackAfterExecution`, for the sake of symmetry.
Compare outputs of those:
```
grep -nRI setCallbackAfter
grep -nRIE 'setAfter.*Callback'
```